jovian/jovian_nb_ext/main.js

```diff
@@ -38,9 +38,22 @@ define([
           const capture_env = window.jvn_params.capture_env;
           const create_new = window.jvn_params.create_new;
           const env_type = window.jvn_params.env_type;
-          const files = window.jvn_params.files;
+          const files =
+            "[" +
+            window.jvn_params.files
+              .split(",")
+              .map(e => "'" + e.trim() + "'")
+              .join(",") +
+            "]";
           var notebook_id;
-          const artifacts = window.jvn_params.artifacts;
+          const artifacts =
+            "[" +
+            window.jvn_params.artifacts
+              .split(",")
+              .map(e => "'" + e.trim() + "'")
+              .join(",") +
+            "]";
+          //window.jvn_params.artifacts;
```

Comment:
You can remove comments which you're not using

Reply:
Sure